ASIM/dev/ASimYaml2ARM/KqlFuncYaml2Arm.py

```diff
@@ -114,7 +114,7 @@
     else:
         logging.warning (f'File or folder "{f}" does not exist.')
 
-files = list(dict.fromkeys(files))  # -- remove duplicates
+files = list(dict.fromkeys(files)).sort()  # -- remove duplicates
```

Comment:
Sort is Important here?

Reply:
Yes
Without the sort, the order differs when run locally and when run as a Github action.

When the user runs the script locally, I want it to create the same files so that when he runs it again on Github, no changes will be made.